test/e2e/deploy/crds.yaml

```
@@ -0,0 +1,24 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: toasters.admiral.submariner.io
+spec:
+  group: admiral.submariner.io
+  version: v1
+  names:
+    kind: Toaster
+    plural: toasters
+  scope: Namespaced
+
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: exportedtoasters.admiral.submariner.io
+spec:
+  group: admiral.submariner.io
+  version: v1
+  names:
+    kind: ExportedToaster
+    plural: exportedtoasters
+  scope: Namespaced
```

```suggestion
  scope: Namespaced
```

(missing newline, not worth fixing on its own but I have other suggestions elsewhere).